src/components/List/components/ListItem.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {block} from '../../utils/cn';
+import {DragHandleIcon} from './DragHandleIcon';
+import type {ListProps, ListItemData, ListSortHandleAlign} from '../types';
+
+const b = block('list');
+
+type ListItemProps<T> = {
+    item: ListItemData<T>;
+    itemIndex: number;
+    active: boolean;
+    selected: boolean;
+    itemClassName?: string;
+    sortable?: boolean;
+    sortHandleAlign?: ListSortHandleAlign;
+    style?: React.CSSProperties;
+    renderItem?: ListProps<T>['renderItem'];
+    onClick?: ListProps<T>['onItemClick'];
+    onMouseMove?: (index: number) => void;
+};
+
+const defaultRenderItem = <T extends unknown>(item: T) => String(item);
```

Comment:
Does `extends unknown` do anything?

Reply:
Yes. It`s avoiding [this](https://stackoverflow.com/a/45576880) trouble